### ISSUE TITLE
Black Border on Buttons When Selected

### ISSUE THREAD

**Issue by ArtKarp** [rwilson504/PCFControls#67]
Button border doesn't use specified color when running app in tab. 

![image](https://user-images.githubusercontent.com/25698082/82744943-d7584280-9d33-11ea-83e7-378871520df8.png)

**Comment by rwilson504** [comment]
Unfortunately this is actually an accessibility of the Chromium browsers and isn't affected by my css.  It used to show up as a blue outline but now they changed it to black.

https://support.google.com/chrome/thread/48974735?hl=en

**Comment by rwilson504** [comment]
Setting the outline for the focused buttons and events to the same color as their parent border colors but setting them at 5px to stand out.
